src/modules/service-catalog/ServiceCatalogItemPage.tsx

```diff
@@ -97,43 +41,99 @@ export function ServiceCatalogItemPage({
   brandId,
 }: ServiceCatalogItemPageProps) {
   const serviceCatalogItem = useServiceCatalogItem(serviceCatalogItemId);
-  const { requestFields, handleChange } = useItemFormFields(
-    serviceCatalogItem,
-    baseLocale
-  );
+  const {
+    requestFields,
+    associatedLookupField,
+    setRequestFields,
+    handleChange,
+  } = useItemFormFields(serviceCatalogItem, baseLocale);
   const { t } = useTranslation();
+  const { addToast } = useToast();
+  const notifyError = useCallback(() => {
```

Comment:
There shouldn't be the need to wrap this function in a `useCallback` hook since it is used only in the `handleRequestSubmit` function

Reply:
ok I removed the useCallback